python_modules/libraries/dagstermill/dagstermill/solids.py

```diff
@@ -216,6 +217,7 @@ def _t_fn(step_context, inputs):
                         )
                         executed_notebook_materialization_path = executed_notebook_path
 
+                    # WTF????? why twice
```

Comment:
are these development artifacts or intended for commit?

Reply:
cleaned up the comments